Manifests/MS/WinPS-Manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Company: Microsoft; Product: Windows Powershell (powershell.exe) and Powershell Core (pwsh.exe) -->

```

Comment:
This should only pertain to PowerShell 7 as the changes required (built upon existing Audit feature in PS7) would be too extensive to backport without a high risk of regression

Reply:
Discussed with Deven, I misunderstood how this setting was going to be used.  This changes what WLDP API returns rather than requiring changes in PS to apply.